apps/retrospective/services/retrospective_feedback.go

```diff
@@ -0,0 +1,254 @@
+package services
+
+import (
+	"github.com/iReflect/reflect-app/apps/retrospective/models"
+	retrospectiveSerializers "github.com/iReflect/reflect-app/apps/retrospective/serializers"
+	userServices "github.com/iReflect/reflect-app/apps/user/services"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"strconv"
+)
+
+// RetrospectiveFeedbackService ...
+type RetrospectiveFeedbackService struct {
+	DB          *gorm.DB
+	TeamService userServices.TeamService
+}
+
+// Add ...
+func (service RetrospectiveFeedbackService) Add(userID uint, sprintID string, retroID string,
+	feedbackType models.RetrospectiveFeedbackType,
+	feedbackData *retrospectiveSerializers.RetrospectiveFeedbackCreateSerializer) (
+	*retrospectiveSerializers.RetrospectiveFeedback,
+	error) {
+	db := service.DB
+
+	retroIDInt, _ := strconv.Atoi(retroID)
+	sprint := models.Sprint{}
+
+	if err := db.Model(&models.Sprint{}).
+		Where("id = ?", sprintID).
```

Comment:
We need to add a check that the Sprint is in `Active` state.

Reply:
done